WooCommerce/Classes/ViewRelated/Orders/Cells/PaymentTableViewCell.swift

```diff
@@ -1,33 +1,33 @@
 import UIKit
 
-class PaymentTableViewCell: UITableViewCell {
+final class PaymentTableViewCell: UITableViewCell {
     @IBOutlet var verticalStackView: UIStackView!
     @IBOutlet var subtotalView: UIView!
-    @IBOutlet public weak var subtotalLabel: UILabel!
-    @IBOutlet public weak var subtotalValue: UILabel!
+    @IBOutlet private weak var subtotalLabel: UILabel!
+    @IBOutlet private weak var subtotalValue: UILabel!
 
-    @IBOutlet public var discountView: UIView!
-    @IBOutlet public weak var discountLabel: UILabel!
-    @IBOutlet public weak var discountValue: UILabel!
+    @IBOutlet private var discountView: UIView!
```

Comment:
Right I had not noticed! It seems like we do not follow any particular convention when it comes to outlets, although we seem to have many more of them declared as just `private var`, without making the weak. 

To be honest, I am afraid to touch anything. If any of those outlets is manipulated at runtime, and I change it to weak, it might crash, I guess?